datacapture/src/main/java/com/google/android/fhir/datacapture/views/QuestionnaireItemEditTextViewHolderFactory.kt

```diff
@@ -76,6 +79,13 @@ internal abstract class QuestionnaireItemEditTextViewHolderDelegate(
           .hideSoftInputFromWindow(view.windowToken, 0)
       }
     }
+
+    textInputEditText.setOnEditorActionListener { view, actionId, _ ->
+      if (actionId != EditorInfo.IME_ACTION_NEXT) {
+        false
+      }
+      view.focusSearch(FOCUS_DOWN)?.requestFocus(FOCUS_FORWARD) ?: false
```

Comment:
why are they different?

```suggestion
      view.focusSearch(FOCUS_FORWARD)?.requestFocus(FOCUS_FORWARD) ?: false
```

Reply:
1. With change `view.focusSearch(FOCUS_FORWARD)?.requestFocus(FOCUS_FORWARD) ?: false`
- it shows warning, please see attached image.
<img width="974" alt="warning" src="https://user-images.githubusercontent.com/86107848/146323869-ccf9c9cd-cf96-4ee0-8bb0-3cb2af766acf.png">
- Yes, focus was given to the focusable item.
   (https://user-images.githubusercontent.com/86107848/146324243-302c3cbe-e68c-4cd3-9c53-37dc23350cb1.mov).


2. With change `view.focusSearch(FOCUS_DOWN)?.requestFocus(FOCUS_FORWARD) ?: false`
-  There is no warning.
-  It scrolls page down.
   (https://user-images.githubusercontent.com/86107848/146324407-33e48607-04cf-44c8-859c-33e21f45c1d3.mov)

Reply:
Updated suggested change.

Reply:
Build fails
/Users/runner/work/android-fhir/android-fhir/datacapture/src/main/java/com/google/android/fhir/datacapture/views/QuestionnaireItemEditTextViewHolderFactory.kt:89: Error: Must be one of: View.FOCUS_LEFT, View.FOCUS_UP, View.FOCUS_RIGHT, View.FOCUS_DOWN [WrongConstant]

Reply:
Interesting, so I guess they: 

- https://newbedev.com/fatal-crash-focus-search-returned-a-view-that-wasn-t-able-to-take-focus
- https://stackoverflow.com/questions/13614101/fatal-crash-focus-search-returned-a-view-that-wasnt-able-to-take-focus/47991577

are probably using deprecated apis/constants.

have you tried using FOCUS_DOWN in both?

Reply:
Updated change `view.focusSearch(FOCUS_DOWN)?.requestFocus(FOCUS_DOWN) ?: false`